python/lsst/ctrl/bps/quantum_clustering_funcs.py

```diff
@@ -123,13 +123,13 @@ def _check_clusters_tasks(cluster_config, task_graph):
         Raised if task label appears in more than one cluster def or
         if there's a cycle in the cluster defs.
     """
-    # Build a PipelineTask graph of just labels because TaskGraph
+    # Build a PipelineTask graph of just labels because PipelineGraph
```

Comment:
The second line of this comment (which was unchanged) should say `NodeKey` instead of `TaskDef`.

FWIW, I didn't use strings as the keys in `make_task_xgraph` because in rare cases a task label is the same as a dataset type name and that would break some related `make_*_xgraph` methods.  I could probably add an option to `make_task_xgraph` and `make_dataset_type_xgraph` to make them do this transformation to regular `str` keys, since in those graphs there wouldn't be any conflicts.  Definitely not worth holding up this ticket, unless it sounds so good you feel motivated to add that to pipe_base yourself.

Reply:
I took a quick look at those functions to see about making the transformation to regular str keys.  But I've run out of time to make changes before this weekly and I prefer getting this ticket merged because it unblocks using clustering in certain cases.   We can revisit the str keys addition in a different ticket.